static/templates/electionCycles.hbs

```diff
@@ -0,0 +1,20 @@
+<fieldset class="js-subcycle-select toggles">
+  {{#each this}}
+  <label>
+    <input
+        type="radio"
+        name="cycle-toggle"
+        value="{{this.max}}"
+        class="panel-toggle-control"
```

Comment:
Oh, I modeled the markup after the toggles on the committee detail page. Do we not need that class there either? I can get rid of them all if they don't do anything.